//---------------------------------------------------------------------
//---------------------------------------------------------------------
syntax = "proto3";

//---------------------------------------------------------------------
//---------------------------------------------------------------------
package ni.measurementlink.measurement.tests.v1;

//---------------------------------------------------------------------
//---------------------------------------------------------------------
option csharp_namespace = "NationalInstruments.MeasurementServices.Measurements";

//---------------------------------------------------------------------
//---------------------------------------------------------------------

message MeasurementParameter{
    float float_data=1;
    double double_data=2;
    int32 int32_data=3;
    uint32 uint32_data=4;
    int64 int64_data=5;
    uint64 uint64_data=6;
    bool bool_data=7;
    string string_data =8;
    repeated double double_array_data=9; 
}
